File: 프로그래머스/연습문제/Lv1/이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    idx=0; answer=""
    for i in range(len(s)):
        if s[i] == ' ':
            answer += ' '
            idx = 0
        elif idx%2 == 1:
            answer += s[i].lower()
            idx += 1
        else:
            answer += s[i].upper()
            idx += 1

    return answer

File: 프로그래머스/연습문제/Lv1/이상한 문자 만들기/이상한 문자 만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    int idx = 0;
    for(int i=0; i<s.length(); i++){
        if(s[i] == ' ') idx = 0;
        else if(idx++%2) s[i] = tolower(s[i]);
        else s[i] = toupper(s[i]);
    }
    
    return s;
}
